docs/admob/android.md

```diff
@@ -37,3 +37,20 @@ public class MainApplication extends Application implements ReactApplication {
   // ...
 }
 ```
+
+## Enable Legacy HTTP Library in AndroidManifest
+
+Android 9.0 removed by default all traces of the old Apache HTTP library. However, it seems that some libraries need it. You need to add this to your manifest inside the <application> element to enable legacy HTTP Library.
+  
+`<uses-library
+    android:name="org.apache.http.legacy"
+    android:required="false"/>`
```

Comment:
The title says to enable this but both the values are set to `false`, this seems incorrect?

Reply:
Indeed. It's pretty subtle, [this is from the MobileSDK google group moderator](https://groups.google.com/forum/#!msg/google-admob-ads-sdk/K1HKru7RutY/VEr-Qm00BwAJ), in contact with their team I would suppose:

> Note: The android:required="false" attribute is required for apps that have a minimum SDK of 23 or lower, because on devices with API levels lower than 24, the org.apache.http.legacy library is not available. (On those devices, the Apache HTTP classes are available on the bootclasspath.)

Imaging the errors and triage on each step of figuring this out --> pain